client/src/components/application/NonlinearStepper/GeneralInfo.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Radio, FormControlLabel, RadioGroup } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import InputText from '../../common/TextField';
+import SelectBox from '../SelectBox';
+import Typography from '../../common/Typography';
+import { GeneralInfoStyles } from './style';
+
+const GereralInfos = ({ handleFormInput, formValues, errorMsg }) => {
+  const classes = GeneralInfoStyles();
+
+  return (
+    <>
+      <div className={classes.gender}>
+        <Typography variant="h5" color="primary">
+          Gender
+        </Typography>
+        <RadioGroup
+          row
+          aria-label="gender"
+          name="gender"
+          onChange={handleFormInput}
+          value={formValues.gender}
+          defaultValue="Female"
+        >
+          <FormControlLabel
+            className={classes.gender}
+            value="female"
+            control={<Radio color="primary" />}
+            label="Female"
+          />
+
+          <FormControlLabel
+            className={classes.gender}
+            value="male"
+            control={<Radio color="primary" />}
+            label="Male"
+          />
+        </RadioGroup>
+      </div>
+      <InputText
+        id="nameID"
+        value={formValues.fullName}
+        placeholder="Enter full name"
+        label="Full Name"
+        name="fullName"
+        onChange={handleFormInput}
+        isError={errorMsg.includes('Full Name is required')}
+        message="ex:- Sam Smith"
+      />
+      <InputText
+        id="mobID"
+        value={formValues.mobileNumber}
+        placeholder="Enter Mobile Number"
+        label="Mobile Number"
+        name="mobileNumber"
+        onChange={handleFormInput}
+        isError={errorMsg.includes('Mobile Nubmer is required')}
+        message="ex:- 05xx xxx xxx"
+      />
+      <SelectBox
+        fullWidth
+        label="Age"
+        name="age"
+        items={['<18', '18-20', '21-25', '26-30', '31-35', '36-40', '40+']}
+        setVal={handleFormInput}
+        value={formValues.age}
+        isError={errorMsg.includes('Age is required')}
+      />
+      <SelectBox
+        fullWidth
+        label="Where do you live in Gaza?"
+        name="address"
+        items={[
+          'North of Gaza Strip',
+          'Gaza City',
+          'Middle Area of Gaza',
+          'Khan Younis',
+          'Rafah',
+        ]}
+        setVal={handleFormInput}
+        value={formValues.address}
+        isError={errorMsg.includes('Address is required')}
+      />
```

Comment:
you should replace `Where do you live in Gaza?` with `Address` for Now and when we accept Login PR, you should check the user's data, if his location `gaza` then you have to display `gaza` areas for him else `khalil`  area, for now let it gaza areas as default, but change the title name. 
```suggestion
      <SelectBox
        fullWidth
        label="Address"
        name="address"
        items={[
          'North of Gaza Strip',
          'Gaza City',
          'Middle Area of Gaza',
          'Khan Younis',
          'Rafah',
        ]}
        setVal={handleFormInput}
        value={formValues.address}
        isError={errorMsg.includes('Address is required')}
      />
```

Reply:
Ok, I will figure it out